roles/custom/matrix-nginx-proxy/defaults/main.yml

```diff
@@ -253,6 +253,10 @@ matrix_nginx_proxy_proxy_grafana_hostname: "{{ matrix_server_fqn_grafana }}"
 matrix_nginx_proxy_proxy_sygnal_enabled: false
 matrix_nginx_proxy_proxy_sygnal_hostname: "{{ matrix_server_fqn_sygnal }}"
 
+# Controls whether proxying the mautrix wsproxy should be done.
+matrix_nginx_proxy_proxy_mautrix_wsproxy_enabled: false
+matrix_nginx_proxy_proxy_mautrix_wsproxy_hostname: "{{ matrix_server_fqn_mautrix_wsproxy }}"
```

Comment:
Adding additional stuff into nginx-proxy is discouraged. Certain new components (like `matrix-sliding-sync`) only contain Traefik support.

Of course, given that nginx support is already available, we can keep it, but.. you don't have to spend time on it if it's not fully working or something.